include/OpenSpaceToolkit/Astrodynamics/Trajectory/State.hpp

```diff
@@ -59,7 +58,23 @@ class State
         const Shared<const CoordinatesBroker>& aCoordinatesBrokerSPtr
     );
 
-    /// @brief                  Constructor.
+    /// @brief                  Constructor. This constructor makes a new Coordinates Broker under the hood for every
+    /// State.
```

Comment:
Maybe add an extra sentence talking about the footprint it leaves and why using the other constructor is encouraged (without making this constructor look bad though 😂 )